filebeat/input/udp/input.go

```diff
@@ -90,9 +104,31 @@ func NewInput(
 		outlet:  out,
 		udp:     udp,
 		started: false,
+		metrics: metrics,
 	}, nil
 }
 
+func configID(config *conf.C) (string, error) {
```

Comment:
I didn't realize that this input is not converted to the [`v2.Input`](https://github.com/elastic/beats/blob/141ad337a4d28f482e6fe0c97c37ad21d54028e2/filebeat/input/v2/input.go#L60) interface that provides an ID. Can you please scope what the effort would be to convert it to the new interface.

Reply:
It looks reasonably straight forward. The only inputs here that are v2 are filestream, journald, kafka and unix. The update would be better in another PR.